.golangci.yaml

```diff
@@ -44,6 +45,286 @@ issues:
+      - pkg: github.com/gardener/gardener/pkg/gardenlet/apis/config/validation
+        alias: gardenletconfigvalidation
+      - pkg: github.com/gardener/gardener/pkg/gardenlet/controller/(v[\w\d]+)
+        alias: ${1}controller
+      - pkg: github.com/gardener/gardener/pkg/healthz
+        alias: gardenerhealthz
+      - pkg: github.com/gardener/gardener/pkg/mock/controller-runtime/manager
+        alias: mockmanager
+      - pkg: github.com/gardener/gardener/pkg/mock/controller-runtime/client
+        alias: mockclient
+      - pkg: github.com/gardener/gardener/pkg/mock/apiserver/authorization/authorizer
+        alias: mockauthorizer
+      - pkg: github.com/gardener/gardener/pkg/mock/client-go/discovery
+        alias: mockdiscovery
+      - pkg: github.com/gardener/gardener/pkg/mock/client-go/core/v1
+        alias: mockcorev1
+      - pkg: github.com/gardener/gardener/pkg/mock/go/io
+        alias: mockio
+      - pkg: github.com/gardener/gardener/pkg/operation/garden
+        alias: gardenpkg
+      - pkg: github.com/gardener/gardener/pkg/operation/seed
+        alias: seedpkg
+      - pkg: github.com/gardener/gardener/pkg/operation/shoot
+        alias: shootpkg
+      - pkg: github.com/gardener/gardener/pkg/operator/client
+        alias: operatorclient
+      - pkg: github.com/gardener/gardener/pkg/operator/features
+        alias: operatorfeatures
+      - pkg: github.com/gardener/gardener/pkg/operator/apis/config/v1alpha1
+        alias: operatorconfigv1alpha1
+      - pkg: github.com/gardener/gardener/pkg/operator/apis/config/validation
+        alias: operatorconfigvalidation
+      - pkg: github.com/gardener/gardener/pkg/registry/core/rest
+        alias: corerest
+      - pkg: github.com/gardener/gardener/pkg/registry/operations/rest
+        alias: operationsrest
+      - pkg: github.com/gardener/gardener/pkg/registry/seedmanagement/rest
+        alias: seedmanagementrest
+      - pkg: github.com/gardener/gardener/pkg/registry/settings/rest
+        alias: settingsrest
+      - pkg: github.com/gardener/gardener/pkg/registry/core/backupbucket/storage
+        alias: backupbucketstore
+      - pkg: github.com/gardener/gardener/pkg/registry/settings/clusteropenidconnectpreset/storage
+        alias: clusteroidcpresetstore
+      - pkg: github.com/gardener/gardener/pkg/registry/core/shoot
+        alias: shootregistry
+      - pkg: github.com/gardener/gardener/pkg/resourcemanager/client
+        alias: resourcemanagerclient
+      - pkg: github.com/gardener/gardener/pkg/resourcemanager/apis/config/v1alpha1
+        alias: resourcemanagerconfigv1alpha1
+      - pkg: github.com/gardener/gardener/pkg/resourcemanager/apis/config/validation
+        alias: resourcemanagerconfigvalidation
+      - pkg: github.com/gardener/gardener/pkg/resourcemanager/predicate
+        alias: resourcemanagerpredicate
+      - pkg: github.com/gardener/gardener/pkg/resourcemanager/controller/(v[\w\d]+)
+        alias: ${1}controller
```

Comment:
Can we do something like,
`gardener/pkg/*/controller`, so that everything which is of the form pkg/<package>/controller is selected, instead of repeating this same two lines for controllermanager, gardenlet, operator, resourcemanager etc..
If it's possible, we can even go for similar optimizations in other alias as well.